.github/workflows/preview-storybook.yml

```
@@ -0,0 +1,89 @@
+        run: |
+          has_releases=$(grep -l "^releases:" .yarn/versions/* | wc -l)
+          if [ "$has_releases" -eq "0" ]; then
+            echo "No release files found, skipping..."
+            echo "should_skip=true" >> $GITHUB_OUTPUT
+            echo "## ⚠️ Storybook preview not available" >> $GITHUB_STEP_SUMMARY
+            echo "Storybook preview is not available for this PR because there are no changes available for a new release." >> $GITHUB_STEP_SUMMARY
+          else
+            echo "should_skip=false" >> $GITHUB_OUTPUT
+          fi
+
+      - uses: ./.github/actions/setup-node-deps
```

actionlint (1.7.4)

description is required in metadata of "Setup Node and Install Dependencies" action at "/home/jailuser/git/.github/actions/setup-node-deps/action.yml"

(action)